src/funding/paypal/template.jsx

```diff
@@ -147,10 +146,52 @@ function ButtonPersonalization(opts : LabelOptions) : ?ChildType {
 }
 
 
+const getTagLineForDivideLogoAnimationExperiment = (opts) => {
```

Comment:
Hmmm should this be named tagline? Tagline normally refers to the text below the button. Could we call it labelText or something like that?